web/cobrands/sass/_waste.scss

```diff
@@ -20,6 +20,10 @@ body.waste {
         font-size: 19px; font-size: 1.1875rem; line-height: 1.31579;
       }
     }
+
+    #form-submit-row.loading {
+        background: url("/cobrands/fixmystreet/images/spinner-white.gif") flip(0%,100%) 100% no-repeat;
```

Comment:
Gets hidden behind the disabled button in the SocietyWorks Showcase cobrand:

<img width="278" alt="Screenshot 2021-03-02 at 14 34 07" src="https://user-images.githubusercontent.com/739624/109666036-b5afa580-7b66-11eb-8d67-bb52206454e9.png">

Reply:
Need this deployed to prevent duplicates, so this can be fixed later